backend/src/config.ts

```diff
@@ -24,6 +24,22 @@ export const DAPP_DID_URI = loadEnv('DAPP_DID_URI') as Kilt.DidUri
 export const DAPP_NAME = process.env.DAPP_NAME ?? 'Web3-Login-Demo'
 export const JWT_SIGNER_SECRET = loadEnv('JWT_SIGNER_SECRET')
 
+// To determine which CType to use, refer to the CType Hash
+const SOCIAL_KYC_EMAIL_CTYPE = 
+'0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914bd9ac' 
+// Required properties for socialKYC  
+export const SOCIAL_KYC_EMAIL_REQUIRED_PORPERTIES = 'Email' 
+// Specify the Social KYC Trusted Attester DID. Credentials issued by this Attester will be accepted.
+export const SOCIAL_KYC_ATTESTER = 
+'did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY' 
+
+// Configerable Credential types 
+export const CTYPE_HASH = loadEnv('CTYPE_HASH') || SOCIAL_KYC_EMAIL_CTYPE 
+export const TRUSTED_ATTESTERS = 
```

Comment:
Same for all these other values. Very good that they are configurable, but I don't see why they would have to have defaults.

Reply:
We can specify what those values are in the README, but the code itself should not assume any defaults. Also, if you check the README, in the list of required env variables there is no mention of these ones, meaning that if I want to change it I would have to dig into the code to know what to change. Listing them in the README would help, and at that point we can say `use 0x......` if you want to verify SKYC email credentials.

Reply:
Antonio's point is well taken. We must remove the defaults. Our users are expected to possess programming knowledge and understand how to use environment variables.

Reply:
removed at [33828af](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/33828afac3d7b6fba9a93fe3882a3603b909daa4)

Reply:
Ok, adding them to the `README` and then `env.example` is also nice.